_posts/2025-04-29-juan-luis-opensource-journey.md

```diff
@@ -0,0 +1,152 @@
+
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/LLciYo3rqTQ" title="A Journey Through Open Source - Webinar by Juan Luis" frameborder="0" allowfullscreen></iframe>
+
+
+> Building on the themes discussed in the webinar, this post explores how open source software has revolutionized technology, collaboration, and innovation over the past seventy years.
+
+
+Open source software has revolutionized technology, collaboration, and innovation over the past seventy years. But its story is much more than a timeline of programming languages and licenses—it's a story about people, ideals, communities, and the evolving meaning of "freedom" in the digital world.
+
+In this blog, we trace the history of open source software, starting from the early days of room-sized computers and academic code sharing, through the founding of the GNU project and the free software movement, the explosive growth of Linux and the World Wide Web in the 1990s, the rise of GitHub and big data in the 2000s, to the complex challenges of sustainability, ethics, and generative AI that define today’s landscape.
+
+This post is based on a webinar hosted by Data Umbrella, a community-funded nonprofit supporting underrepresented voices in data science. Featuring a deep and thoughtful talk by Juan Luis—product manager at QuantumBlack AI by McKinsey and a longtime advocate for the PyData community—we’ll walk through the key milestones, cultural shifts, and philosophical debates that have shaped open source into what it is today.
+
+Whether you're new to open source or a seasoned contributor, this journey offers insights not just into the software we use every day, but into the values, struggles, and future possibilities of the open technology movement.
+
+## The Genesis of Computing and Hacker Ethos in the 1950s and 60s
+
+Let's begin at the dawn of computing, exploring what we might call ["the rise and fall of hacker culture"](https://www.youtube.com/watch?v=LLciYo3rqTQ&t=569s) – a subjective framing, as emphasized by Juan.  So as I said I'm going to start with the very beginning of the history of computing. I call this session the rise and fall of hacker culture and just to make it very clear that this is a subjective perspective... The 1950s and 60swere a world away from today's computing landscape. Programming involved punch cards and other "arcane methods" to input programs into room-sized computers like the PDP-10, a prominent machine in research and university settings.  So we're talking about the [50s and 60s](https://www.youtube.com/watch?v=LLciYo3rqTQ&t=595s) and computing was nothing like we can imagine it now. Programmers were using punch cards and very arcane methods to type their programs into the computers and such computers were enormous devices that took the place of a full room almost. This photograph is from a PDP-10 which was one of the famous computers that could be found in research departments and at universities and so on.
```

Comment:
I think "programmers were using punch cards and very arcane methods" as well as mentions of the PDP-10 are sort of repeated twice?